src/app/pages/ArticlePage/PagePromoSections/utility/index.js

```diff
@@ -0,0 +1,10 @@
+export default (sectionType, assetUri, uri, contentType, promoIndex = 0) => {
```

Comment:
this needs a test

Reply:
Will Do.

Reply:
nvm, back in the code. I went for the a11yId approach because the approach i tested was not working